autopush/web/webpush.py

```diff
@@ -43,7 +44,9 @@ def post(self, api_ver="v1", token=None):
             headers=self.valid_input["headers"],
             ttl=self.valid_input["headers"]["ttl"]
         )
-
+        self._client_info["uaid"] = hasher(user_data.get("uaid"))
+        self._client_info["channel_id"] = user_data.get("chid")
+        self._client_info["stack"] = ["%s.post" % self.__module__]
```

Comment:
one last thing -- let's *always* put this in _client_info by making BaseHandler._init_info do it, which is called at the beginning of every request.  Make it something like '%s.%s (%s)' % (self.__module__, self.__name__, self.request.method) i.e. 'autopush.endpoint.EndpointHandler (POST)'

the name 'stack' is confusing w/ the raven stack arg Ben pointed out above, maybe make it "handler" or something